src/main/java/seedu/address/logic/commands/DeleteModuleCommand.java

```diff
@@ -0,0 +1,53 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.commands.CommandWords.DELETE_MODEL;
+import static seedu.address.logic.commands.CommandWords.MODULE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_MODULE;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.module.Module;
+
+/**
+ * Deletes a module identified using it's module code.
+ */
+public class DeleteModuleCommand extends Command {
+
+    public static final String COMMAND_WORD = MODULE;
```

Comment:
The command word should actually be = CommandWords.MODULE + " " + CommandWords.DELETE_MODEL
Then the usage will just need to be COMMAND_WORD